src/test/kotlin/com/hivemq/extension/gradle/ConfigurationCacheTest.kt

```diff
@@ -94,7 +94,7 @@ internal class ConfigurationCacheTest {
                 <priority>0</priority>
                 <start-priority>1000</start-priority>
             </hivemq-extension>
-            """.trimIndent(),
+            """.trimIndent() + "\n",
```

Comment:
```suggestion

            """.trimIndent(),
```
minor comment: in multi-line strings if you want to have a newline at the end you need to add an empty line.